modules/node-shared/src/main/scala/org/tessellation/node/shared/http/routes/SnapshotRoutes.scala

```diff
@@ -40,78 +42,119 @@ final case class SnapshotRoutes[F[_]: Async: Hasher: KryoSerializer, S <: Snapsh
   implicit def binaryAndJsonEncoders[A <: AnyRef: Encoder]: List[EntityEncoder[F, A]] =
     List(BinaryCodec.encoder[F, A], CirceEntityEncoder.circeEntityEncoder[F, A])
 
+  private val serviceUnavailableNodeNotReady: F[Response[F]] =
+    ServiceUnavailable(("message" ->> s"Node should be in Ready state before fetching") :: HNil)
```

Comment:
Maybe:
"Node not yet ready to handle snapshot API requests" 🤔 
I would not expose the internal implementation e.i. `Ready` state

Reply:
@TheMMaciek I think it's important to give info when the client can expect to make a request and success 🤔 Instead of saying explicitly about the state, it can say `Node is not ready yet` or sth like that. Would that be okay for you @TheMMaciek?
cc: @IPadawans